Library/src/Systems/Settings/SettingsSystem.cpp

```diff
@@ -937,15 +937,13 @@ void SettingsSystem::GetAvatarInfo(AvatarInfoResultCallback Callback)
 
 		InternalResult.SetAvatarType(static_cast<AvatarType>(type.GetInt()));
 
-		if (Json.HasMember("identifierType") && Json["identifierType"].IsInt()
-			&& static_cast<VariantType>(Json["identifierType"].GetInt()) == VariantType::Integer)
+		if (Json.HasMember("identifierType") && static_cast<VariantType>(Json["identifierType"].GetInt()) == VariantType::Integer)
```

Comment:
[Question] Do we still need this now the type is explicitly csp::common::String?

Reply:
For Get we do, because it may have previously been stored as an int, and we need this to be backward compatible